Dockerfile

```diff
@@ -26,7 +26,8 @@ RUN npm run build
 #
 # Note: This stage builds the base image for production. Presently we are not
 # using this on the production site, but only use it as base for the dev build.
-FROM python:3.9.9-slim as base
+# Pin "bullseye" as it matches Ubuntu 20.04 -- Heroku 20 stack currently used in production.
+FROM python:3.9.9-slim-bullseye as base
```

Comment:
This is a related change but a good one to make. We should pin the Debian version to avoid surprises in the future.